UHResidentInformationAPI/V1/Factories/EntityFactory.cs

```diff
@@ -10,13 +16,48 @@ public static ResidentInformation ToDomain(this Person databaseEntity)
         {
             return new ResidentInformation
             {
-                HouseReference = databaseEntity.houseRef,
-                PersonNumber = databaseEntity.personNo,
+                HouseReference = databaseEntity.HouseRef,
+                PersonNumber = databaseEntity.PersonNo,
                 FirstName = databaseEntity.FirstName,
                 LastName = databaseEntity.LastName,
                 DateOfBirth = databaseEntity.DateOfBirth.ToString("O", CultureInfo.InvariantCulture),
                 NINumber = databaseEntity.NINumber
             };
         }
+
+        public static List<ResidentInformation> ToDomain(this IEnumerable<Person> people)
+        {
+            return people.Select(p => p.ToDomain()).ToList();
+        }
+
+        public static Address ToDomain(this DbAddress databaseEntity)
+        {
+            return new Address
+            {
+                PropertyRef = databaseEntity.PropertyRef,
+                AddressLine1 = databaseEntity.AddressLine1.Trim(),
+                PostCode = databaseEntity.PostCode
+            };
+        }
+
+        public static Phone ToDomain(this TelephoneNumber databaseEntity)
+        {
+            var canParseType = Enum.TryParse<PhoneType>(databaseEntity.Type, out var type);
+            return canParseType ? new Phone
```

Comment:
Will there always be a parsable phone type? Maybe we should be returning an object with `PhoneNumber` and `LastModified` and a `null` for `Type` if the type isn't parsable? I'm not sure though because I don't know what the data is like.

Reply:
That's true. 
I could add some code to account for if the phone type is null. 
I just had a look at the schema again and the column for this is nullable.

Reply:
Changes made in d235b521d286dc4ae1b83b279c4bbe2ec5eaa7e0